README.md

```diff
@@ -59,5 +60,10 @@ blank.
 welcomes new contributors to the repository. If `contributing` is not
 present, the [Rust contributing.md][rustcontrib] will be linked instead. 
 
+If PRs should be filed against a branch other than `master`, specify the
+correct destination in the `expected_branch` field. If `expected_branch` is
+left out, highfive will assume that PRs should be filed against `master` and
+post a warning if they're against some other branch.
```

Comment:
nit: this sounds like highfive will only warn if the branch is master, but it will warn in any case.